src/utils/traffic-mesh.js

```diff
@@ -48,6 +52,8 @@ const startForwardProxy = async ({ port, frameworkPort, functionsPort, publishDi
     `http://localhost:${frameworkPort}`,
     '--watch',
     publishDir,
+    '--bundler',
+    EDGE_HANDLERS_BUNDLER_CLI_PATH,
```

Comment:
Just making sure: will this work on Windows? Specifically, inside `cmd.exe`, which does not handle shebangs, i.e. `/path/to/cli.js` does not work, but `node /path/to/cli.js` does.

Reply:
Good catch! Somebody's alert 😉

It will work due to https://github.com/netlify/traffic-mesh/pull/1190

Reply:
Yes, that works :+1: